src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1514,7 +1514,6 @@ private MethodSymbol FindEntryPoint(CancellationToken cancellationToken, out Imm
                 {
                     entryPoint = viableEntryPoints[0];
                 }
-
                 viableEntryPoints.Free();
```

Comment:
>viableEntryPoints.Free(); [](start = 16, length = 25)

Please keep the empty line. Since this is the only change in the file, consider reverting it to the original state.